anime_downloader/watch.py

```diff
@@ -122,11 +122,45 @@ def _append_to_watch_file(self, anime):
 
         self._write_to_watch_file(data)
 
-    def _write_to_watch_file(self, animes):
-        animes = [anime.__dict__ for anime in animes]
+    def _write_to_watch_file(self, animes, MAL_import = False):
+        if not MAL_import:
+            animes = [anime.__dict__ for anime in animes]
+
         with open(self.WATCH_FILE, 'w') as watch_file:
             json.dump(animes, watch_file)
 
+    def _import_from_MAL(self,PATH):
+        import xml.etree.ElementTree as ET #Standard Library import, conditional as it only needs to be imported for this line
+        root = ET.parse(PATH).getroot()
+        list_to_dict = []
+        values = { 'Plan to Watch' : { 'planned' : 'yellow' },
+                   'Completed' : { 'completed' : 'green' },
+                   'Watching' : { 'watching' : 'blue' },
+                   'Dropped' : { 'dropped' : 'red' }
+                   }
+        for type_tag in root.findall('anime'):
+            a = type_tag.find('my_watched_episodes').text
+            b = type_tag.find('my_score').text
+            c = type_tag.find('my_status').text
+            colour = str(list(values[c].values())[0])
+            c = str(list(values[c].keys())[0])
+            d = type_tag.find('series_title').text
+            e = type_tag.find('series_episodes').text
+            list_to_dict.append( {
+                "episodes_done": int(a),
+                "_timestamp": 0,
```

Comment:
Best to use the acutal timestamp in case we want to do cool stuff with that in the future.

Reply:
We can't do a timestamp though when we initialise an anime at 0, unless I don't know what timestamp does

Reply:
Im pretty sure it's just unix time, generating that on import shouldn't be an issue. I don't know why it's used, but best to keep it consistent.

Reply:
Nevertheless, all the timestamps would still be near identical due to the speed of this method. Fixing them all at zero is pretty much isomorphic. Unless you want to start randomly generating timestamps(?)

Reply:
Originally, I thought timestamps was how long in seconds media has been watched. But it's def not that, I guess it's time created?

Reply:
I'm pretty sure it's Unix time when added (possibly updated). Could be used to automatically update old generations, idk.

Reply:
Then surely there's no need to set a value. Remember, a user has to go into an anime and then re set the provider, this will generate the correct timestamp.

Reply:
Done.